docs/requirements.txt

```diff
@@ -0,0 +1,14 @@
+numpy>=1.20.0
+mdanalysis>=2.0.0
+ipython
+pickleshare # ipython savefigs
+MDAnalysisTests
+MDAnalysisData
+networkx # LeafletFinder dependency
+tidynamics # MSD dependency
+sphinx <7.0
```

Comment:
I know we pinned this for a bit, but I think we've finally been able to unpin Sphinx and work with the most recent versions. Is `sphinx<7.0` still needed here?

Reply:
Thanks, actually now I realized I can fully skipped `Sphinx` here because `sphinx>=6.2.1` is coming as transient dep from `mdanalysis-sphinx-theme` anyway. :)  I'll delete line 9 here.